net-libs/gwenhywfar/gwenhywfar-5.10.2.recipe

```diff
@@ -0,0 +1,130 @@
+SUMMARY="Multi-platform helper library for security applications and libraries"
+DESCRIPTION="OS abstraction functions for various projects"
+HOMEPAGE="http://www.aqbanking.de"
+COPYRIGHT="Martin Preuss"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/aqbanking/gwenhywfar/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="2b0b9dd49b95f025f4e0c0346ba88e20b893407c444fccf6403a4da349954f04"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="79.10.2"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	gwenhywfar$secondaryArchSuffix = $portVersion
+	lib:libgwenhywfar$commandSuffix = $libVersionCompat
+	lib:libgwengui_cpp$commandSuffix = $libVersionCompat
+	lib:libgwengui_gtk3$commandSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libatk_1.0$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
+	lib:libcairo_gobject$secondaryArchSuffix
+	lib:libgcrypt$secondaryArchSuffix
+	lib:libgdk_pixbuf_2.0$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgnutls$secondaryArchSuffix
+	lib:libgpg_error$secondaryArchSuffix
+	lib:libgtk_3$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libpango_1.0$secondaryArchSuffix
+	lib:libpangocairo_1.0$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	gwenhywfar${secondaryArchSuffix}_devel = $portVersion
+	cmd:gwenhywfar_config$commandSuffix
+	devel:libgwenhywfar$secondaryArchSuffix = $libVersionCompat
+	devel:libgwengui_cpp$secondaryArchSuffix = $libVersionCompat
+	devel:libgwengui_gtk3$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	gwenhywfar$secondaryArchSuffix == $portVersion base
+	devel:libgcrypt$secondaryArchSuffix
+	devel:libgnutls$secondaryArchSuffix
+	devel:libgtk_3$secondaryArchSuffix
+	"
+
+SUMMARY_tools="Command line utilities for gwenhywfar"
+PROVIDES_tools="
+	gwenhywfar${secondaryArchSuffix}_tools = $portVersion
+	cmd:gct_tool$commandSuffix
+	cmd:gsa$commandSuffix
+	cmd:gwbuild$commandSuffix
+	cmd:mklistdoc$commandSuffix
+	cmd:typemaker$commandSuffix
+	cmd:typemaker2$commandSuffix
+	cmd:xmlmerge$commandSuffix
+	"
+REQUIRES_tools="
+	haiku$secondaryArchSuffix
+	gwenhywfar$secondaryArchSuffix == $portVersion base
+	lib:libgnutls$secondaryArchSuffix
+	lib:libgpg_error$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libgcrypt$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libgcrypt$secondaryArchSuffix
+	devel:libgnutls$secondaryArchSuffix
+	devel:libgtk_3$secondaryArchSuffix
+    "
+BUILD_PREREQUIRES="
+	cmd:autoreconf
+	cmd:gettext
```

Comment:
Nitpicking (and no reason to not merge this), I've changed the way gettext is packaged, does it really search for "cmd:gettext" or does configure check for msgfmt/msgmerge? Or even with gettext/xgettext also?